l10n_br_base/models/inherited_mail_template.py

```diff
@@ -0,0 +1,229 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2016 Taŭga Tecnologia
+#   Aristides Caldeira <aristides.caldeira@tauga.com.br>
+# License AGPL-3 or later (http://www.gnu.org/licenses/agpl)
+#
+
+from __future__ import division, print_function, unicode_literals
+
+
+import logging
+
+from odoo import api, fields, models, tools, _
+from odoo.exceptions import UserError, ValidationError
+from odoo.addons.mail.models.mail_template import (
+    mako_safe_template_env,
+    mako_template_env,
+    format_date,
+    format_tz,
+)
+from ..constante_tributaria import *
```

Comment:
this is for the SPED or product based NFE, this should hence go to the l10n_br_account_product module instead